das-dashboard/backend/shared/utils/remote_scp.py

```
@@ -0,0 +1,79 @@
+        key_path = profile.get("profile_ssh_keypath") or DEFAULT_SSHKEY_CLONE_PATH
+
+        if not os.path.exists(key_path):
+            raise ValueError("SSH key is not configured. Set up your profile first.")
+
+        return username, key_path
+
+    def connect(self, host: str) -> SSHClient:
+        username, key_path = self.ensure_profile()
+
+        ssh = SSHClient()
+        ssh.set_missing_host_key_policy(AutoAddPolicy())
```

Ruff (0.15.18)

[error] Paramiko call with policy set to automatically trust the unknown host key

(S507)